bounce.py

```diff
@@ -390,6 +394,19 @@ def _button_release_cb(self, win, event):
                     self._start_step()
         return True
 
+    def _mouse_motion_cb(self, widget, event):
+        if self.ball.grabbed and self._fraction is not None:
+            if event.is_hint:
+                mouse_x = event.window.get_pointer()[0]
```

Comment:
On line 400, `mouse_x` is set but never used.  (Line 404 is indented within the false branch of the conditional test at line 399.)